src/redux/reducers/experimentSettings/processingConfig/loadedProcessingConfig.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-param-reassign */
+import produce, { current } from 'immer';
+import _ from 'lodash';
+
+import initialState from '../initialState';
+
+// Will be removed once the default values are stored in the pipeline
+const withAutomaticSettings = (processingConfig) => {
```

Comment:
This was added originally in a different PR than this one because we add defaultFilterSettings property in the ui on receiving processingConfig, it should probably be moved to the api or (even better) to gem2s when generating the processingConfig